roles/cinder-data/tasks/main.yml

```diff
@@ -22,12 +22,15 @@
             dest=/etc/cinder/cinder.encryption.conf mode=0640
             owner=root group=cinder
   notify: restart cinder services
-  when: cinder.fixed_key is defined
+  when:
```

Comment:
I'll confirm with Niraj if we need to remove fixed_key or move it into cinder.encryption.